docs/src/development/development_setup.md

```diff
@@ -119,6 +120,31 @@ for further details.
 We are currently moving to follow [Blue Style Guide](https://github.com/invenia/BlueStyle).
 All PRs should follow this style guide.
 
+To set up the formatter:
+
+- Go to the VS Code settings panel or press `ctrl+,`.
+- Search for "Julia Formatter"
+
+One of the top matches, if not the first match, should be the configuration flags for the
+formatter.
+
+```bash
+Julia-format: Flag
+Flag(s) for JuliaFormatter. See Formatting Options and options.jl.
+```
+
+The team has adopted the following configuration options for auto-formatting:
+
+```bash
+style=BlueStyle(), indent=4, margin=256, always_for_in=true, for_in_replacement="∈", whitespace_typedefs=true, import_to_using=true, align_struct_field=true, align_assignment=false, align_conditional=true, align_pair_arrow=false, normalize_line_endings="unix", align_matrix=true, join_lines_based_on_source=true, indent_submodule=true, surround_whereop_typeparameters=false, yas_style_nesting=true
+```
+
+Next, search for `trim` and ensure the following options are all ticked/enabled:
+
+- Files: Trim Final Newlines
+- Files: Trim Trailing Whitespace
+- Editor: Trim Auto Whitespace
+
```

Comment:
```Suggestion

or, as suggested by Blue Style Guide, append the following to your settings.json file

{
    "[julia]": {
        "editor.detectIndentation": false,
        "editor.insertSpaces": true,
        "editor.tabSize": 4,
        "files.insertFinalNewline": true,
        "files.trimFinalNewlines": true,
        "files.trimTrailingWhitespace": true,
        "editor.rulers": [92],
    },
}
```

Are we using margins of 256 instead of 92?

Reply:
The settings UI just edits the settings.json file.

We are using 92 character limit, and I do indeed have that set as a VS Code option, not a Julia Formatter option. I'll update.

Reply:
Done

Reply:
Note, I've opted not to include instructions to edit the settings.json file directly.

Of course, if people are comfortable doing so, that's fine, but given the Julia Formatter options are equivalent and we may adjust these in future, I think best to just give one set of canonical instructions.